src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -182,20 +187,57 @@ export class CalciteTipManager {
     this.nextTip();
   };
 
+  tipManagerKeyDownHandler = (event: KeyboardEvent): void => {
+    switch (event.key) {
+      case "ArrowUp":
+        event.preventDefault();
```

Comment:
What was up doing?

Reply:
native behavior is it scrolls the page up/down when you press the arrow keys. This prevents the page from scrolling when focused on the manager.